app/views/questionnaires/_questionnaire_debug.html.slim

```diff
@@ -1,4 +1,4 @@
 - if ENV['RAILS_ENV'].eql?('development')
   pre.debug_dump
-    = @questionnaire.debug_summary
-    = @questionnaire.module_item.debug_summary
\ No newline at end of file
+    = @questionnaire.module_item.debug_summary
```

Comment:
```ruby
  pre.debug_dump
    = @questionnaire.module_item.debug_summary
    = @questionnaire.debug_summary
```

Reply:
Do you mean replace the first line with the second line? If so, some of the information stops displaying:
<img width="985" alt="image" src="https://user-images.githubusercontent.com/13471320/189598769-b0352d5f-b600-4ad0-ba4d-5e756bef36a1.png">

Reply:
No have both so they are appended and the `Questionnaire` only adds info the `ModuleItem` doesn't